solarized-theme-utils.el

```diff
@@ -24,7 +24,7 @@
 ;;
 
 ;;;; Code:
-(require 'cl)
+(require 'cl-lib)
```

Comment:
We also need to add a dep to `cl-lib` from GNU ELPA for older Emacsen (e.g. 24.1).